package.json

```diff
@@ -25,25 +25,25 @@
   },
   "homepage": "https://github.com/pnarielwala/gulp-sloc2",
   "dependencies": {
-    "mkdirp": "^0.4.0",
-    "lodash": "^2.4.1",
-    "sloc": "^0.1.11",
-    "event-stream": "^3.1.2",
-    "gulp-util": "^2.2.14"
+    "gulp-util": "^3.0.8",
+    "lodash": "^4.17.11",
+    "mkdirp": "^0.5.1",
+    "sloc": "^0.2.1"
   },
   "devDependencies": {
-    "coveralls": "^2.10.0",
-    "event-stream": "^3.1.2",
+    "coveralls": "^3.0.2",
+    "event-stream": "^4.0.1",
     "expect.js": "^0.3.1",
-    "gulp": "^3.6.2",
-    "gulp-jshint": "^1.5.5",
-    "gulp-load-plugins": "^0.5.0",
-    "gulp-mocha": "^0.4.1",
-    "gulp-plumber": "^0.6.1",
-    "istanbul": "^0.2.7",
-    "jshint-stylish": "^0.1.5",
-    "mocha": "^1.18.2",
-    "mocha-lcov-reporter": "0.0.1",
-    "strip-ansi": "^0.2.0"
+    "gulp": "^3.9.1",
```

Comment:
I'd like to fix the npm audit errors around gulp too, but that's much more of a breaking change to upgrade to gulp 4.